service/policy/objects.proto

```diff
@@ -308,11 +308,6 @@ message PublicKey {
       expression: "this.matches('^https://[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?(\\\\.[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?)*(/.*)?$')"
     }];
 
-    // public key; PEM of RSA public key; prefer `cached`
```

Comment:
You'll need to reserve this number:

https://protobuf.dev/programming-guides/proto3/#reserved-field-numbers

Reply:
@dmihalcik-virtru Could you take another look at this?